components/atoms/Checkbox/checkbox.tsx

```diff
@@ -10,27 +10,29 @@ interface CheckboxProps extends React.ComponentPropsWithoutRef<typeof CheckboxPr
 
 const Checkbox = React.forwardRef<React.ElementRef<typeof CheckboxPrimitive.Root>, CheckboxProps>(
   ({ className, label, id, ...props }, ref) => {
-    const getId = () => (id ? id : label?.replaceAll(" ", "_").toLowerCase());
+    const getId = () => (id ? id : label?.replace(/\s+/g, "-").toLowerCase());
```

Comment:
Why this change?
```suggestion
    const getId = () => (id ? id : label?.replace(" ", "-").toLowerCase());
```

Reply:
@OgDev-01 for better readability and alignment of styles.

Reply:
@OgDev-01, I cannot understand why the test is failing.